bucket/jar2app.json

```diff
@@ -0,0 +1,12 @@
+{
+  "homepage": "https://github.com/Jorl17/jar2app",
+  "version": "master",
+  "license": "GPL-2.0-only",
+  "url": "https://github.com/Jorl17/jar2app/archive/master.zip",
```

Comment:
Do you really need whole repository? Isn't file jar2app.py enough?

Reply:
After just a quick look, it seems those files in `jar2app_basefiles` might be used:
https://github.com/Jorl17/jar2app/blob/49eb2c5fc29b7239486b3756d185a1d63e1f97ab/jar2app.py#L303-L313

Reply:
@Ash258  I don't know so I played it safe... 
Anyway it's only 29K so I don't think it's an issue.